release_bot/releasebot.py

```diff
@@ -43,14 +45,16 @@ def __init__(self, configuration):
         self.logger = configuration.logger
         # FIXME: it's cumbersome to work with these dicts - it's unclear how the content changes;
         #        get rid of them and replace them with individual variables
-        self.new_release = {}
-        self.new_pr = {}
+        self.new_release = New_Release()
+        self.new_pr = New_PR()
 
     def cleanup(self):
-        if 'tempdir' in self.new_release:
-            self.new_release['tempdir'].cleanup()
-        self.new_release = {}
-        self.new_pr = {}
+        # if 'tempdir' in self.new_release:
+        #     self.new_release['tempdir'].cleanup()
+        # What is the use of the above statements? There is no reference of tempdir
+        # in the entire code.
```

Comment:
You're right, it's some left over, remove it, please.

Reply:
Cleared.